### ISSUE TITLE
Questions about Kiiti training

### ISSUE THREAD

**Issue by deepcharle** [lmb-freiburg/flownet2#199]
Thank you for your excellent work in optical flow. Note that the groundtruth flow values in Kiiti is very sparse, how do you deal with this case when you augment the image pairs and corresponding optical flow to keep it from interpolating with nonsense zero optical flow values?

**Comment by nikolausmayer** [comment]
If I remember correctly, we do not use geometric augmentation for KITTI. I'm pretty sure that our data augmentation cannot deal with NaN pixels.

The KITTI setting is so tightly constrained that any change to the data bears a significant risk of destroying the advantage you hope to gain by finetuning. Stretching, warping, or rotating images makes no sense because the KITTI test set has none of these effects.

**Comment by deepcharle** [comment]
thank you very much for your reply
